### DISCUSSION TITLE
How to show connection name in a tab with ERD

### DISCUSSION THREAD

**Discussion by jacekgajek** [dbeaver/dbeaver#17990] [Q&A]
I have several connections to different databases (15 and counting, welcome to microservices).

The SQL script tabs are like `<foo_db [qa]> Script-5`
How can I add info about connection to the tab with an ERD diagram? Currently it's like this:
![image](https://user-images.githubusercontent.com/68021034/197157996-15247f89-d1b9-40be-a219-d79ed5737236.png)
I'd like to have eg `<foo_db [qa]> public` where "foo_db [qa]" is a connection name.

OR

have tab grouped by connection, like this:
```
connection_1                                       connection_2
console        erd       some_table                console         erd        some_table
```
However this is MUCH harder and was already discusesd here. Basically Eclipse team is working on it since late 2002 and in 2014 they said that they don't have enough resources. https://bugs.eclipse.org/bugs/show_bug.cgi?id=28722

OR

allow me to open more tabs in the "schema" view
![image](https://user-images.githubusercontent.com/68021034/197161759-c227e872-8ce1-444c-8842-71492dd541bd.png)

**Comment by EurekaJiang** [comment]
when you hover over the tab, you can see the  connection info , so I think it's the width that's the problem  when  many same named tabs  are opened at the same time ,even though they're with connection name.     maybe in the "schema" view add a sql console tab is  better
